Applications/PaintBrush/main.cpp

```diff
@@ -96,7 +97,27 @@ int main(int argc, char** argv)
         return;
     }));
 
-    menubar->add_menu("Edit");
+    auto& edit_menu = menubar->add_menu("Edit");
+
+    auto paste_action = GUI::CommonActions::make_paste_action(
+        [&](const GUI::Action&) {
+            auto old_bitmap = RefPtr<Gfx::Bitmap>(paintable_widget.bitmap());
+            auto clipboard_bitmap = GUI::Clipboard::the().get_bitmap(old_bitmap->format(), old_bitmap->size());
+            if (!clipboard_bitmap)
+                return;
+            auto new_bitmap = Gfx::Bitmap::cloned_from(clipboard_bitmap);
+            paintable_widget.set_bitmap(*new_bitmap);
```

Comment:
Going forward, pasting shouldn't just replace what I'm working on.
Instead, it should become a new layer that can be moved around etc. :)

Reply:
Yeah, however that's a quite expensive feature on PaintBrush right now and I wanted to focus my less than optimal focus and knowledge on this particular task. However the next thing should be having at least support for 2 layers on PaintBrush, mspaint style :)